src/aspire/classification/averager2d.py

```diff
@@ -334,24 +364,16 @@ def __init__(
         src,
         alignment_basis=None,
         n_angles=360,
-        n_x_shifts=1,
-        n_y_shifts=1,
+        radius=None,
         num_procs=1,
         dtype=None,
     ):
         """
         See AligningAverager2D and BFRAverager2D, adds: `n_x_shifts`, `n_y_shifts`.
```

Comment:
Docstring mentions `n_x_shifts` and `n_y_shifts` which no longer exist.

Reply:
Good catch, fixed, thanks!